code/style.css

```diff
@@ -1,4 +1,211 @@
-html {
-  background: #0025ff;
-  color: #fff;
-}
+html {
+  background: rgb(233, 197, 203);
+  color: black;
+  font-family: 'Open Sans', sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+/****HEADER****/
+.header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.headline p, h1 {

```

Comment:
I'm not sure if that's what you wanted to achieve - if you would like to apply `text-align: center` value to both `p` and `h1` nested inside `.headline`, then you need to fix you reference to this one :
```
.headline p, .headline h1 {
  text-align: center;
}
```